### ISSUE TITLE
HPO name search breaks with colon in string

### ISSUE THREAD

**Issue by stolpeo** [varfish-org/viguno#159]
**Describe the bug**
The name search returns a 500 error when the string contains a `:`. It seems to interpret the first part of the string up to the `:` as a field.

```
## Input: "HP:"
curl 'http://localhost:8000/proxy/varfish/viguno/hpo/terms?name=HP%3A'
Error parsing query: Syntax Error: HP: 
```

```
## Input: "HP:00"
curl 'http://localhost:8000/proxy/varfish/viguno/hpo/terms?name=HP%3A00'
Error parsing query: Field does not exist: 'HP'
```

**Comment by stolpeo** [comment]
Actually I don't know if it is intended to also search by term id, but the error is strange anyway.

**Comment by holtgrewe** [comment]
The full-text search engine interprets this in the query. Will be fixed in #173.
